### PR TITLE
Migrate from Travis to GitHub Actions

### PR DESCRIPTION
**Pull request by JafarAbdi**

### CODE REVIEW

File: .github/workflows/industrial_ci_action.yaml [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    strategy:
+      matrix:
+        env:
+          - ROS_DISTRO: foxy
+            ROS_REPO: main
+            CCOV: true
+          - ROS_DISTRO: foxy
+            ROS_REPO: testing
+    env:
+      AFTER_SETUP_UPSTREAM_WORKSPACE: 'vcs pull $BASEDIR/upstream_ws/src'
```
Comment (tylerjw) [inline .github/workflows/industrial_ci_action.yaml:24]: I don't think you need this line.

```suggestion
```

File: .github/workflows/industrial_ci_action.yaml [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: cache upstream_ws
+        uses: pat-s/always-upload-cache@v2.1.3
+        with:
+          path: ${{ env.BASEDIR }}/upstream_ws
+          key: upstream_ws-${{ env.CACHE_PREFIX }}-${{ hashFiles('moveit2.repos') }}-${{ github.run_id }}
+          restore-keys: |
+            upstream_ws-${{ env.CACHE_PREFIX }}-${{ hashFiles('moveit2.repos') }}
```
Comment (tylerjw) [inline .github/workflows/industrial_ci_action.yaml:43]: I don't see any upstream workspace defined, no need to cache it.

```suggestion
```

**Review by tylerjw — APPROVED** [review]
A couple minor things.  Approved so you can just merge it.

File: README.md [hunk 3]
```diff
@@ -20,7 +20,9 @@ Developed by [Dave Coleman](http://dav.ee) at the Correll Robotics Lab, Universi
 
 <a href='https://ko-fi.com/A7182AMW' target='_blank'><img height='36' style='border:0px;height:36px;' src='https://az743702.vo.msecnd.net/cdn/kofi2.png?v=0' border='0' alt='Buy Me a Coffee at ko-fi.com' /></a>
 
- * [![Build Status](https://travis-ci.org/PickNikRobotics/rviz_visual_tools.svg)](https://travis-ci.org/PickNikRobotics/rviz_visual_tools) Travis CI
+ * [![Format](https://github.com/1PickNikRobotics/rviz_visual_tools/actions/workflows/format.yml/badge.svg?branch=foxy-devel)](https://github.com/PickNikRobotics/rviz_visual_tools/actions/workflows/format.yaml?branch=foxy-devel)
```
Comment (JafarAbdi) [inline README.md:23]:
```suggestion
 * [![Format](https://github.com/PickNikRobotics/rviz_visual_tools/actions/workflows/format.yml/badge.svg?branch=foxy-devel)](https://github.com/PickNikRobotics/rviz_visual_tools/actions/workflows/format.yaml?branch=foxy-devel)
```

**Review by tylerjw — APPROVED** [review]
Do you mind rebasing this into 2 commits?

* Migration to GitHub Actions
* Fix formatting errors

Then we can merge this with a normal merge and it will keep those two commits separate?

**Pull request merged**